.gitmodules

```diff
@@ -1,4 +1,4 @@
-[submodule "src/realm-core"]
-	path = src/realm-core
-	url = https://github.com/realm/realm-core
+[submodule "packages/realm_dart/src/realm-core"]
```

Comment:
Can the native bits live outside of `realm_dart` or do they have to be in a subfolder? Not a strong opinion, but I mildly prefer to have the native bits - both core and our custom native code - outside of the dart packages.

Reply:
I was thinking about that as well.

The long term goal here is to move to use the new native-asset feature. They suggest keeping the native src, in `<package>/src` and the dart src in `<package>/lib/src`, so I'm inclined to stick with that.

However, I do intent to move all dart code in `realm_dart`, except the bindings (`native/realm_bindings.dart`) to `realm` and have `realm_dart` be a simple shim around the native code... Once native-assets land.

Would that sit with you?

Reply:
I would prefer that we don't ship the native source code with the dart packages since it's unusable, which is why I'd like to see it live outside the packages. Even with native assets, I'd probably prefer that we ship just the prebuilds unless we believe there's a use case where someone might want to build the library locally.

Reply:
The src won't be in the package on pub.dev. It isn't today either. It will just be a submodule you can call in with git - if you need to build the native assets. For end users it will just be downloaded from S3.